examples/src/demo.cr

```diff
@@ -45,14 +45,30 @@ MY_APP_SERVER.config do |app|
   # (HTTP METHODS)[https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html]
   routes :static do
     # Each route is defined as follow
-    # verb, resources : String, controller : Symbol, action : Symbol,
-    # pipeline : Symbol
-    get "/*", StaticController, :index
+    # verb resource : String, controller : Symbol, action : Symbol
+    get "/index.html", StaticController, :index
   end
 
+  # Routes accepts a pipeline name and a scope a pipeline represents a stach of
+  # http handlers that will process the current request
   routes :web, "/v2" do
-    get "/hello", HelloController, :index
-    get "/hello/:planet", HelloController, :world
+    # You can also define all resources at once with the resources macro.
+    # This will define the following routes
+    # resoources path, controller, actions
```

Comment:
Short answer is no. I think that should be up to the developer. Implicitly pluralizing I am not a fan of. What is your though on that?